hexa/plugins/connector_iaso/views.py

```diff
@@ -0,0 +1,76 @@
+import uuid
+from logging import getLogger
+
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import get_object_or_404, render
+from django.utils.translation import gettext_lazy as _
+
+from .datacards import FormCard, IASOCard
+from .datagrids import ObjectGrid
+from .models import IASOAccount
+
+logger = getLogger(__name__)
+
+
+def datasource_index(request: HttpRequest, datasource_id: uuid.UUID) -> HttpResponse:
```

Comment:
To keep consistent with other modules, this would be `datasource_detail` or `account_detail`